Translations/pt_BR.po

```diff
@@ -1023,133 +1021,128 @@ msgstr "Enviar Mensagem"
 
 #: Source/control.cpp:348
 msgid "Available Commands:"
-msgstr ""
+msgstr "Comandos Disponíveis:"
 
 #: Source/control.cpp:356
 msgid "Command "
-msgstr ""
+msgstr "Comando "
 
 #: Source/control.cpp:356
 msgid " is unkown."
-msgstr ""
+msgstr " não identificado."
 
 #: Source/control.cpp:359 Source/control.cpp:360
-#, fuzzy
-#| msgid "Description:"
 msgid "Description: "
-msgstr "Descrição:"
+msgstr "Descrição: "
 
 #: Source/control.cpp:359
 msgid ""
 "\n"
 "Parameters: No additional parameter needed."
 msgstr ""
+"\n"
+"Parâmetros: Nenhum parâmetro adicional é necessário."
 
 #: Source/control.cpp:360
-#, fuzzy
-#| msgid "Players: "
 msgid ""
 "\n"
 "Parameters: "
-msgstr "Jogadores: "
+msgstr ""
+"\n"
+"Parâmetros: "
 
 #: Source/control.cpp:380 Source/control.cpp:412
 msgid "Arenas are only supported in multiplayer."
-msgstr ""
+msgstr "Arenas só estão disponíveis no modo multijogador."
 
 #: Source/control.cpp:385
-#, fuzzy
-#| msgid "Are you sure you want to buy this item?"
 msgid "What arena do you want to visit?"
-msgstr "Tem certeza de que quer comprar este item?"
+msgstr "Qual arena você quer visitar?"
 
 #: Source/control.cpp:393
 msgid "Invalid arena-number. Valid numbers are:"
-msgstr ""
+msgstr "Número de arena inválido. Os números válidos são:"
 
 #: Source/control.cpp:399
 msgid "To enter a arena, you need to be in town or another arena."
 msgstr ""
+"Para entrar em uma arena, você precisa estar na cidade ou em outra arena."
 
 #: Source/control.cpp:436
 msgid "Inspecting only supported in multiplayer."
-msgstr ""
+msgstr "Inspeção só está disponível no modo multijogador."
 
 #: Source/control.cpp:441 Source/control.cpp:730
 msgid "Stopped inspecting players."
-msgstr ""
+msgstr "Parar de inspecionar os jogadores."
 
 #: Source/control.cpp:451
 msgid "Inspecting player: "
-msgstr ""
+msgstr "Inspecionar jogador: "
 
 #: Source/control.cpp:460
 msgid "No players found with such a name"
-msgstr ""
+msgstr "Nenhum jogador encontrado com este nome"
 
 #: Source/control.cpp:524
 msgid "/help"
-msgstr ""
+msgstr "/help"
```

Comment:
> Do we need to translate when the text is the same?

Technically no, since gettext will use English text if TL is missing, but IMO string should be still filled in, for sake of completeness, even if it's just and copy-paste

> Should these commands even be translatable?

No, this is a command line parameter, it's not translatable since it doesn't (and shouldn't) change with language

Reply:
> Technically no, since gettext will use English text if TL is missing, but IMO string should be still filled in, for sake of completeness, even if it's just and copy-paste

We'll agree to disagree then. Duplicating the value on every language seems extremely redundant and unnecessary to me.

> No, this is a command line parameter, it's not translatable since it doesn't (and shouldn't) change with language

Why is it in the translation file at all, then? Shouldn't this only contain localizable text? If the cmdline arguments are not supposed to be localized, they should be hardcoded and not go through the localization code path at all.